mocked_server/grpc_trace/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:14
```

Comment:
for now, we have three mocked_server,
1. https mocked server
2. grpc metric mocked server
3. grpc trace mocked server.

can we create a folder named `mocked_servers` and subfolders for each mocked server. 

then, in the imagebuild folder, we make some modification to loop through all the subfolder under `mocked_servers`, build them and push them to ecr with its name as the image tag.  just like how we do for sample_apps: https://github.com/aws-observability/aws-otel-test-framework/blob/terraform/terraform/imagebuild/main.tf#L38

after that, we can define a var `mocked_server` in common.tf just like `sample_app` https://github.com/aws-observability/aws-otel-test-framework/blob/terraform/terraform/imagebuild/main.tf#L38,  use `mocked_server` if `mocked_server_image` is not provided. and get the actual mocked_server image link base on the `mocked_server` value just like how we do for sample apps here: https://github.com/aws-observability/aws-otel-test-framework/blob/terraform/terraform/basic_components/outputs.tf#L45

Reply:
we might also define a var for mocked_server_port since different mocked_server would use different port.

Reply:
will that be possible we can all use 443 for the grpc port, so that we don't need to specify the port?